jquery.flot.absRelTime.js

```diff
@@ -44,32 +62,6 @@ The formatted values look like in the example bellow:
 |Relative Time|0, 300, 600|00:00:00, 00:05:00, 00:10:00|
 |Relative Time|300, 600, 900|00:00:00, 00:05:00, 00:10:00|
 
-formatString usage
-------------------
-There are two flavors of formats you can provide for the date and time formatting options through the
-'formatString' option, and can be mixed between the date and time sections. The first is a simple string
-to indicate date and/or time should be formatted according to the language preference of the browser:
-#d: date
-#T: time
-
-A user could specify formatString = "#d" to indicate the display of a particular date-time should
-just show the date. A string of "#d#T" indicates both date and time should be displayed. Note that time
-is always displayed before the date, and when both are displayed they will be separated by a newline (so order
-of the time and date format substrings doesn't matter).
-
-The second flavor is a more verbose set of options, allowing a user to specify certain characteristics of both
-the date and time displays. The following shows the complete set of options:
-dd/MM/yy(yy) hh:mm:ss.SSS  (NOTE: The (yy) just indicates you can optionally specify a 4 digit year. So 'yy' or 'yyyy'.)
-
-Acceptable date configutations are:
-dd/MM</yy(yy)>  <...> indicates optional part of formatString
```

Comment:
I was using `<...>` to indicate optional values, as you did here (to avoid ambiguity with choices between required alternatives `(...|...)`).  Are you trying to say every format above is optional?